### PR TITLE
Rework dashboard view controllers.

### LINKED ISSUE #293: Strange rendering in dashboard
img_0001

### PR DESCRIPTION
**Pull request by mathebox**
- Fix table view headers.

Fixes #293

### CODE REVIEW

File: xikolo-common/ViewHelpers/TableViewResultsControllerDelegateImplementation.swift [hunk 1]
```diff
@@ -174,18 +178,35 @@ protocol TableViewResultsControllerConfiguration {
...
+    func shouldShowHeader() -> Bool
+
+}
+
+extension TableViewResultsControllerConfiguration {
+
+    func shouldShowHeader() -> Bool {
+        return true
+    }
+
```
Comment (mathebox) [inline xikolo-common/ViewHelpers/TableViewResultsControllerDelegateImplementation.swift]: @bjrne a protocol extension has done the trick ;)

**Comment by jprberlin** [comment]
Ok for me, will wait for @bjrne to review

**Comment by jprberlin** [comment]
@bjrne ok for you?

File: xikolo-common/ViewHelpers/TableViewResultsControllerDelegateImplementation.swift (hunk 1 above)
Comment (bjrne) [inline xikolo-common/ViewHelpers/TableViewResultsControllerDelegateImplementation.swift]: I sometimes wonder wether there is a "protocols" sign in wavy letters above your bed ;P Nice!

**Comment by bjrne** [comment]
Yes, looks fine.

**Comment by mathebox** [comment]
@bjrne What's wrong about having a separate file for this view?

**Comment by bjrne** [comment]
That this is the only place we do this. If we do this everywhere, I'd be more content with it. But I think we have similar cases where we have the view in the storyboard and that would be the moment where I'd be wondering where to find the section header, because it is the only view outside of our storyboards at the moment.

**Pull request merged**